site/viamguides/src/pomodoro-bot/pomodoro-bot.md

```diff
@@ -0,0 +1,439 @@
+<!-- ------------------------ -->
+## Integrate Google Calendar
+
+Duration: 5
+
+Now let's set up your Pomodoro Bot to work with Google Calendar, so you can stay on top of your schedule without losing focus. Once integrated, your bot will automatically sync with your calendar to manage events and reminders.
+
+### Set Up a Google Cloud Project
+
+1. Go to the [Google Cloud Console](https://console.cloud.google.com/).
+2. Create a new project specifically for this integration.
+   ![new project](assets/googleCloud.png)
+3. Once the project is ready:
+    - Open the [API Library](https://cloud.google.com/apis/docs/overview).
+    - Search for "Google Calendar API".
+      ![calendar API](assets/calendarAPI.png)
+    - Click "Enable" to activate the API.
+      ![enable API](assets/enableAPI.png)
+   
+
+### Create a Service Account
+
+1. In the Google Cloud Console, navigate to **IAM & Admin** > **Service Accounts**.
+![IAM & Admin](assets/IAM.png)
+
+2. Set up a new service account:
+    * Create a new service account
+      ![Create Service Account](assets/createServiceAccount.png)
+      ![Name Service Account](assets/nameServiceAccount.png)
+    * Assign the role "Editor" or "Owner" to give it the necessary permissions.
+      ![Assign Roles](assets/assignRoles.png)
+    * Create and download the key file in JSON format. This file will be essential later.
+      ![Manage Keys](assets/manageKeys.png)
+      ![Create Key](assets/createKey.png)
+      ![Download Key](assets/downloadKey.png)
+
+    * Keep the file in a secure location.
+
+### Share Your Calendar with the Service Account
+
+1. Open Google Calendar.
+   ![Google Calendar](assets/googleCalendar.png)
+2. Select the calendar you want to use.
+   ![Select Calendar](assets/selectCalendar.png)
+3. In **Settings and Sharing**, find the "Share with specific people" section:
+   ![Share With Specific People](assets/shareSettings.png)
+    * Enter the email address from the service account key file.
+    * Give it permission to "Make changes to events."
+      ![Share Access](assets/shareAccess.png)
+    * Scroll down to "Integrate Calendar" and copy the **Calendar ID** for the next steps.
+      ![Calendar ID](assets/calendarID.png)
+
+### Connect Your Calendar to the Pomodoro Bot
+
+1. Open the Viam app and go to the **CONFIGURE** tab.
+2. Add the **Calendar module**, and provide:
+   ![Google Calendar Service](assets/addGoogleCalendar.png)
+    * The **Calendar ID** you copied earlier.
+    * The path to the service account JSON file.
+      ![Calendar Configuration](assets/calendarConfig.png)
+3. Save your changes to complete the setup.
+
+### Test the Integration
+
+1. Create a test event in Google Calendar:
+    * Name it something like "Test Meeting" and save the event.
+      ![Test Meeting](assets/testMeeting.png)
+2. Go back to the Viam app and open the **Control** tab.
+      ![Control Tab](assets/controlTab.png)
+3. Use the command interface to check for upcoming calendar events.
```

Comment:
Please include a JSON code block with the command to make it easier to copy into the Do Command field in the app:

```suggestion
3. Use the command interface to check for upcoming calendar events using the following command.
   ```json
   {
     "get_events": {
       "max_results": 10
     }
   }
   ```
```